e2e/cypress/integration/channel/header_spec.js

```diff
@@ -95,7 +95,7 @@ function updateAndVerifyChannelHeader(prefix, header) {
     }
 
     // * Check if channel header description has ellipsis
-    cy.get('#channelHeaderDescription').find('p').
+    cy.get('#channelHeaderDescription > .header-description__text').find('p').
```

Comment:
failed due to recent change - https://github.com/mattermost/mattermost-webapp/pull/4931/files#diff-e14c8484702ecc907896bc1f55f5e56bR458